pinot-broker/src/main/java/org/apache/pinot/broker/routing/RoutingManager.java

```diff
@@ -82,7 +83,7 @@
  * TODO: Expose RoutingEntry class to get a consistent view in the broker request handler and save the redundant map
  *       lookups.
  */
-public class RoutingManager implements ClusterChangeHandler {
+public class RoutingManager implements RouteManager, ClusterChangeHandler {
```

Comment:
Let's name the interface `RoutingManager` and rename the implementation to `DefaultRoutingManager`